kraken/plugins/pod_plugin.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python
+import re
+import sys
+import time
+import typing
+from dataclasses import dataclass, field
+import random
+from datetime import datetime
+from traceback import format_exc
+
+from kubernetes import config, client
+from kubernetes.client import V1PodList, V1Pod, ApiException, V1DeleteOptions
+from arcaflow_plugin_sdk import validation, plugin, schema
+
+
+def setup_kubernetes(kubeconfig_path):
```

Comment:
Is this something we can put in the plugin/__init__ file? so that other plugins will all use it?